test/specs/common/tests/02.column.js

```diff
@@ -0,0 +1,194 @@
+exports.execute = function(options) {
+    var sampleValues = {
+        date: '2016-05-02',
+        timestamptz: '2011-05-06T08:25:25-07:00'
+    };
+
+    describe('About the Column class, ', function() {
+        var schemaName1 = "common_schema_1",
+            schemaName2 = "common_schema_2";
+        var table1_schema1, // has two outbound fks to table2_schema1. with foreign-key annotation.
+            table2_schema1, // has outbound fk to table1_schema2.
+            table1_schema2, // has all the column types we support
+            catalog;
+
+        beforeAll(function(done) {
+            catalog = options.catalog;
+            table1_schema1 = catalog.schemas.get(schemaName1).tables.get('table_1_schema_1');
+            table2_schema1 = catalog.schemas.get(schemaName1).tables.get('table_2_schema_1');
+            table1_schema2 = catalog.schemas.get(schemaName2).tables.get('table_1_schema_2');
+            done();
+        });
+
+        // Test Cases:
+        describe('a column in Table class, ', function() {
+            var column;
+            beforeAll(function() {
+                column = table1_schema2.columns.getByPosition(2);
+            });
+
+            it('should be defined.', function() {
+                expect(column).toBeDefined();
+            });
+
+            it('should have a .formatvalue property.', function() {
+                expect(column.formatvalue).toBeDefined();
+            });
+
+            describe('the .formatvalue property, ', function() {
+                var formatUtils = options.includes.ermRest._formatUtils;
+
+                it('should be empty string if column value is null.', function() {
+                    var column = table1_schema2.columns.getByPosition(2);
+                    expect(column.formatvalue(null)).toEqual(jasmine.any(String));
+                    expect(column.formatvalue(null)).toBe('');
+                });
+
```

Comment:
I don't understand this question. I don't test for an `undefined` column value in these cases.